### PR TITLE
Portfolio: setup and mobile-first

### PR DESCRIPTION
**Pull request by fmanimashaun**
# Portfolio: setup and mobile-first

- The project file structure was setup 
- The nav-bar was created with related styling
- The mobile and desktop nav-bar HTML structures were added but left hidden with my defined CSS rule by the default (*toggle functionality will be added in later projects via javascript* )
- The hero section was created via HTML layout and styling with related content as designed in the Figma file

*My layout styling approach is based on [Every layout](https://every-layout.dev/) principle*

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+
+<body>
+    <!-- ***********************  PAGE OVERALL CONTENT WRAPPER *********************** -->
+    <article class="container">
+
+        <!-- ***********************  MOBILE VIEWPORT MENU  *********************** -->
+        <nav class="mobile-menu-modal close-menu">
```
Comment (nedjwab) [inline index.html:20]: [ ] kindly use a `header` tag for the main site navigation.[ check this article ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header)👀

File: index.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                <article class="hero-heading">
+                    <h1 class="hero-heading-intro">Hey there. I'm Engr. Animashaun</h1>
+                    <h2 class="hero-heading-title">I'm a software developer</h2>
+                </article>
+                <article class="hero-content">
+                    <p class="hero-content-text">I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+                    see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+                </article>
+                <article class="hero-social">
```
Comment (nedjwab) [inline index.html]: [ ] kindly use a `ul` tag for The list of icons. and `li` per icon, each icon has a link a` tag `.The links should be your social media links.🤓

File: README.md [hunk 3]
```diff
@@ -4,12 +4,24 @@
 
 ## Overview
 
-> XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+The project is focus on create a professional portfolio site to showcase my professional skills and experience while highlighting some of my real-life projects and other professional contributions to give an insight into my professional capacity and uniqueness. The Live site can be view from [my personal site](https://fmanimashaun.com/).
```
Comment (nedjwab) [inline README.md:8]: [ ] kindly add a usage section to the README file, to explain how we set up and run the project locally.👀

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @fmanimashaun 

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.
- Good design 🔥🔥🔥

## (Changes Required) Aspects to improve: ♻️
You did great implementing  the project requirements, but you still have some issues: 
 
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (fmanimashaun) [reply index.html:20]: # Main  site navigation issue

- The main site navigation is within the header already
- What you are referencing is the mobile menu modal HTML structured which has been referenced to the container div
- I have added additional comments in the HTML file for clarity

File: README.md (hunk 3 above)
Comment (fmanimashaun) [reply README.md:8]: # Resolved the social media HTML structure
- Thanks for the feedback, it has been implemented as recommended.

File: index.html (hunk 2 above)
Comment (fmanimashaun) [reply index.html]: # Resolved the social media HTML structure
- Thanks for the feedback, it has been implemented as recommended.

File: README.md (hunk 3 above)
Comment (fmanimashaun) [reply README.md:8]: # Usage section
- Well, in my view, I feel it is not needed as the project is a static website (personal portfolio). 
- But if you insist, I might reconsider my stance on it (Cheers).

File: index.html (hunk 1 above)
Comment (mgmediaweb) [reply index.html:20]: Kindly, I recommend you follow the suggestions of the previous reviewer, I enclose a basic structure of what a header should look like

carbon (6)

File: index.html (hunk 2 above)
Comment (mgmediaweb) [reply index.html]: Good job following the advice of the previous reviewer

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @fmanimashaun , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost1

 ## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (fmanimashaun) [reply index.html:20]: # Header structure

- As stated earlier, the header is well structured with the main site navigation within the header element (*see index.html for reference*)
- I have adjusted the logo section as recommended and referenced it to the hero section for return navigation back to the top (*will be implemented in the future projects with javascript*

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @fmanimashaun, I'm @MaganaAsiati, your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :grinning:  :grinning:  :grinning: 
- [ ] [OPTIONAL] I suggest you remove the comment message  inside your code and unneeded file and folder ( js ) it will make your code look clear and more readable  :grinning:  :grinning:  :grinning: 



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏

**Pull request merged**